soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -338,138 +338,138 @@ void InitTrickNames() {
-     Text{"Carpenter's Saw", "Scie du charpentier", "Sierra del carpintero"},
-     Text{"Poacher's Sword", "Hache du chasseur", "Espada del capataz"},
+     Text{"Ancient Bladesaw", "Longue épée archéonique", "Mandoble ancestral"},
+     Text{"Woodcutter's Axe", "Hache de bûcheron", "Hacha de leñador"},
      Text{"Grog's Saw", "Scie de Grog", "Sierra del Cazador Furtivo"}};
   trickNameTable[GI_SWORD_BROKEN] = {
-     Text{"Broken Biggoron's Sword", "Épée brisée de Grogoron", "Espada de Biggoron rota"},
-     Text{"Broken Giant's Knife", "Lame des Géants brisée", "Daga gigante rota"},
-     Text{"Biggoron's Sword", "Épée de Biggoron", "Espada de Biggoron"}};
+     Text{"Broken Noble Sword", "Épée noble brisée", "Espada noble rota"},
+     Text{"Broken Picori Blade", "Épée Minish brisée", "Espada minish rota"},
+     Text{"Decayed Master Sword", "Épée de légende pourrie", "Espada decadente de leyenda"}};
   trickNameTable[GI_PRESCRIPTION] = {
-     Text{"Biggoron's Prescription", "Ordonnance de Grogoron", "Receta de Biggoron"},
-     Text{"Eyedrop Prescription", "Ordonnance de gouttes", "Receta ocular"},
-     Text{"Urgent Prescription", "Ordonnance urgente", "Prescripción"}};
+     Text{"Swordsman's Scroll", "Précis d'escrime", "Esgrimidorium"},
+     Text{"Portrait of Oren", "Portrait d'Orlène", "Retrato de Oren"},
+     Text{"Letter to King Zora", "Lettre au roi Zora", "Carta al Rey Zora"}};
   trickNameTable[GI_FROG] = {
      Text{"Don Gero", "Don Gero", "Don Gero"},
-     Text{"Eyedrop Frog", "Grenouille-qui-louche", "Globo Ocular de Rana"},
-     Text{"Frog", "Crapaud", "Rana"}};
+     Text{"Hot-Footed Frog", "Grenouille à pieds chauds", "Rana de patas calientes"},
+     Text{"Lost Swordsmith", "Forgeron perdu", "Espadachín perdido"}};
   trickNameTable[GI_EYEDROPS] = {
-     Text{"Biggoron's Eyedrops", "Gouttes de Grogoron", "Gotas de Biggoron"},
-     Text{"Hyrule's Finest Eyedrops", "Eau du Lac Hylia", "Gotas oculares"},
+     Text{"Moon's Tear", "Larme de Lune", "Lágrima de Luna"},
+     Text{"Engine Grease", "Graisse moteur", "Grasa del motor"},
      Text{"Zora Perfume", "Parfum Zora", "Perfume Zora"}};
   trickNameTable[GI_CLAIM_CHECK] = {
-     Text{"Clay Check", "Certificat Grogoron", "Comprobante de Reclamación"},
+     Text{"Ancient Tablet", "Stèle ancienne", "Litografía arcana"},
      Text{"Sheikah Slate", "Tablette Sheikah", "Piedra Sheikah"},
      Text{"Cyclone Slate", "Ardoise des tornades", "Pizarra de los Torbellinos"}};
 
   trickNameTable[GI_SKULL_TOKEN] = {
-     Text{"Skulltula Token", "Bon de Skulltula dorée", "Símbolo de Skulltula"},
-     Text{"Golden Skulltula Spirit", "Pièce de Skulltula dorée", "Tóken de Skulltula Dorada"},
-     Text{"Gold Walltula Token", "Jeton de Walltula dorée", "Skulltula dorada"}};
+     Text{"Maiamai", "Ti'gorneau", "Maimai"},
+     Text{"Gratitude Crystal", "Cristal de gratitude", "Gema de gratitud"},
+     Text{"Korok Seed", "Noix korogu", "Semilla de kolog"}};
```

Comment:
These ones were forcing the player to pay better attention, feels better to have the old ones